schema/ast.go

```diff
@@ -164,6 +164,11 @@ type Sequence struct {
 	OwnedBy     string
 }
 
+type DefaultDefinition struct {
+	name  string
+	value *Value
```

Comment:
Does `DEFAULT NULL` define a default constraint? If it's not the case and `DEFAULT NULL` is absolutely the same as writing nothing there, I thought we don't need to make this a pointer (nullable) because `defaultDef` itself is also a pointer (nullable).

Reply:
Yes, NULL defines the default constraint like other value.